go.mod

```
@@ -8,9 +8,9 @@ require (
 	github.com/docker/docker v26.1.1+incompatible
 	github.com/google/uuid v1.6.0
 	github.com/testcontainers/testcontainers-go v0.30.0
-	golang.org/x/exp v0.0.0-20240416160154-fe59bbe5cc7f
+	golang.org/x/exp v0.0.0-20240506185415-9bf2ced13842
 	gotest.tools/v3 v3.5.1
-	nhooyr.io/websocket v1.8.10
+	nhooyr.io/websocket v1.8.11
```

Version 1.8.11 of nhooyr.io/websocket causes an error during client close.
```suggestion
	nhooyr.io/websocket v1.8.10
```